api/src/pdc/services/auth/providers/OIDCProvider.ts

```
@@ -73,31 +88,87 @@ export class OidcProvider implements InitHookInterface {
+    >(token, {
+      ignoreExpiration: true,
+    });
+    /**
+     * Handle V1 token format conversion
+     */
+    if ("id" in payload && "app" in payload) {
+      payload.v = 1;
+      payload.a = payload.app!;
+      payload.o = payload.id!;
+      payload.s = "operator";
+      delete payload.id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)